hera_sim/data/tests/test_sim_file.yaml

```diff
@@ -1,5 +1,5 @@
 #diffuse_foreground:
-#    Tsky_mdl: HERA_Tsky_mdl['xx']
+#    Tsky_mdl: # XXX how to give a hook to a npz that Travis can find???
```

Comment:
If you're doing a test, you can maybe write your yaml dynamically as a string, write it to a temp file, and then read it in. That way you know where the npz is. Also remember to delete all test artifacts at the end of your test! Using temporary files is useful for this!

Reply:
Thanks for the advice! I'll rewrite the tests to use temporary files.